mongo/options/clientencryptionoptions.go

```diff
@@ -43,7 +82,47 @@ func MergeClientEncryptionOptions(opts ...*ClientEncryptionOptions) *ClientEncry
 		if opt.KmsProviders != nil {
 			ceo.KmsProviders = opt.KmsProviders
 		}
+		if opt.TLSConfig != nil {
+			ceo.TLSConfig = opt.TLSConfig
+		}
 	}
 
 	return ceo
 }
+
+// Used by both ClientEncryptionOptions.SetTLSOptions() and AutoEncryptionOptions.SetTLSOptions()
+func applyTLSOptions(tlsOpts map[string]map[string]interface{}, tlsConfigs map[string]*tls.Config) (map[string]*tls.Config, error) {
+	for provider, opts := range tlsOpts {
+		cfg := &tls.Config{MinVersion: tls.VersionTLS12}
```

Comment:
I remember some offline conversation about this minimum version. Can we leave a comment justifying why we enforce it?

Reply:
Done!